db/db_compaction_test.cc

```diff
@@ -5974,6 +5974,18 @@ TEST_F(DBCompactionTest, CompactionWithBlob) {
             blob_file->GetTotalBlobBytes());
   ASSERT_EQ(compaction_stats[1].num_output_files, 1);
   ASSERT_EQ(compaction_stats[1].num_output_files_blob, 1);
+
+  TablePropertiesCollection all_table_props;
+  ASSERT_OK(db_->GetPropertiesOfAllTables(&all_table_props));
+
+  ASSERT_EQ(all_table_props.size(), 1);
+
+  const auto& table_props = all_table_props.begin()->second;
+  ASSERT_NE(table_props, nullptr);
+
+  const auto& user_props = table_props->user_collected_properties;
+  ASSERT_NE(user_props.find(TablePropertiesNames::kBlobFileMapping),
+            user_props.end());
```

Comment:
Why don't also check the value of property?

Reply:
I felt that would be redundant since we have dedicated unit tests for that. Here, I'm just trying to make sure BlobDB's property collector was active as expected.